src/object.d

```diff
@@ -4156,15 +4156,19 @@ void __ctfeWrite(scope const(char)[] s) @nogc @safe pure nothrow {}
  * Create RTInfo for type T
  */
 
-template RTInfoImpl(size_t[] pointers)
+template RTInfoImpl(size_t[] pointerBitmap)
 {
-    immutable size_t[pointers.length] data = pointers[];
-    immutable RTInfoImpl = data.ptr;
+    immutable size_t[pointerBitmap.length] RTInfoImpl = pointerBitmap[];
```

Comment:
I don't remember exactly, but there might have been some troubles with this version when first implemented 2015. Looks good now.